### PR TITLE
[DOCS] New ADR proposal for patch version support

### PR DESCRIPTION
**Pull request by donaldheppner**
New ADR proposal for patch version support

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code

### CODE REVIEW

**Review by Kilo59 — APPROVED** [review]
This is great!

File: docs/adr/0002-support-latest-point-release.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Date: 2023-03-24
+
+## Status
+
+Proposed
+
+## Context
+
+Dependency management is an ongoing challenge when publishing a library. While our constraints need to be reasonably loose, we must narrow the scope of what we support (both with our test suite and in community support channels) to avoid overwhelming our automated testing resources and support teams.
+
```
Comment (Kilo59) [inline docs/adr/0002-support-latest-point-release.md:12]: 💯

File: .gitignore [hunk 2]
```diff
@@ -146,3 +146,7 @@ metastore_db
 # dependency management
 Pipfile
 Pipfile.lock
+
+# recommended venv directories from our contributor guide
+gx_dev
+.gx_dev
```
Comment (Kilo59) [inline .gitignore]:
```suggestion
.gx_dev

```

**Comment by anthonyburdi** [comment]
Thank you for this!!

**Pull request merged**